rfc6962-bis.xml

```diff
@@ -906,23 +979,20 @@ an accepted root certificate.
               </list>
             </t>
             <t>
-              The <spanx style="verb">hash</spanx> must be calculated as defined in <xref target="tree"/>. The <spanx style="verb">tree_size</spanx> must designate an existing v2 STH. Because of skew, the front-end may not know the requested STH. In that case, it will return the latest STH it knows, along with an inclusion proof to that STH. If the front-end knows the requested STH then only <spanx style="verb">leaf_index</spanx> and <spanx style="verb">audit_path</spanx> are returned.
+              The <spanx style="verb">hash</spanx> must be calculated as defined in <xref target="tree_leaves"/>. The <spanx style="verb">tree_size</spanx> must designate an existing v2 STH. Because of skew, the front-end may not know the requested STH. In that case, it will return the latest STH it knows, along with an inclusion proof to that STH. If the front-end knows the requested STH then only <spanx style="verb">inclusion</spanx> is returned.
             </t>
             <t hangText="Outputs:">
               <list style="hanging">
-                <t hangText="leaf_index:">
-                  The 0-based index of the entry corresponding to the <spanx style="verb">hash</spanx> parameter.
-                </t>
-                <t hangText="audit_path:">
-                  An array of base64 encoded Merkle Tree nodes proving the inclusion of the chosen certificate.
+                <t hangText="inclusion:">
+                  A base64 encoded <spanx style="verb">TransItem</spanx> of type <spanx style="verb">inclusion_proof</spanx> whose <spanx style="verb">audit_path</spanx> array of Merkle Tree nodes proves the inclusion of the chosen certificate in the selected STH.
```

Comment:
So, s/audit_path/inclusion_path/ then?

Then we'd have "consistency_proof" and "inclusion_proof" for the TransItem types; "consistency" and "inclusion" for the API outputs; and "consistency_path" and "inclusion_path" for the NodeHash vectors.